waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -102,3 +102,10 @@ genPrismaSchema wasp = createTemplateFileDraft dstPath tmplSrcPath (Just templat
     entityToPslModelSchema entity =
       Psl.Generator.Model.generateModel $
         Psl.Ast.Model.Model (Wasp.Entity._name entity) (Wasp.Entity._pslModelBody entity)
+
+genMigrationsDir :: Wasp -> Maybe FileDraft
+genMigrationsDir wasp =
+  getMigrationsDir wasp >>= \waspMigrationsDir ->
+    Just $ createCopyDirFileDraft (SP.castDir projectMigrationsDir) (SP.castDir waspMigrationsDir)
+  where
+    projectMigrationsDir = dbRootDirInProjectRootDir </> dbMigrationsDirInDbRootDir
```

Comment:
```suggestion
    genProjectMigrationsDir = dbRootDirInProjectRootDir </> dbMigrationsDirInDbRootDir
```
I have a tendency to use "generated project" (or `genProject` etc) instead of just "project" but I see that then again we have `dbRootDirInProjectRootDir` (so without `gen` prefix). Any thoughts @Martinsos , what do you suggest/prefer?

Reply:
Commented in the previous feature -> terrible naming by me! In this case I would keep it as it is for consistency until we do global rename effort, but in that comment above I would add "generated".

Reply:
Yeah the whole project vs generated project vs blah is all up in the air. I can add generated in these contexts you noted! 👍🏻